app/Http/Controllers/Users/UserController.php

```
@@ -21,43 +20,35 @@ final class UserController extends Controller
                 ]),
         ]);
     }
 
     public function show(User $user): Response
     {
-        return Inertia::render('settings/users/show', [
+        return Inertia::render('users/show', [
             'user' => $user,
-            'role' => $user->roles->first(),
-            'permissions' => $user->permissions,
+            'role' => $user->roles->first()->name,
```

PHPStan (2.0.3)

Cannot access property $name on App\Models\Role|null.


(property.nonObject)